packages/react/src/EditorContent.tsx

```diff
@@ -164,10 +164,14 @@ export class PureEditorContent extends React.Component<
 
     this.initialized = false
 
-    if (!editor.isDestroyed) {
-      editor.view.setProps({
-        nodeViews: {},
-      })
+    try {
```

Comment:
I wonder if, instead of handling the error with a try-catch block, an extra check could be performed on the editor view to determine if the `setProps` method can be called.

For example, this property: https://prosemirror.net/docs/ref/#view.EditorView.isDestroyed

But the solution of the try-catch block works too.

Reply:
The problem with that was that there's no actual editor view to check on in the first place. The thrown error actually comes from a check if the editor view is accessible or not (that's used for using Tiptap on the server where you would not have an EditorView).

https://github.com/ueberdosis/tiptap/blob/next/packages/core/src/Editor.ts#L302

Maybe there's another way but I think just catching the error to proceed with the editor component logic should be fine.

Reply:
This PR would have actually fixed this issue: https://github.com/ueberdosis/tiptap/pull/6233

`this.editorView` stored the actual view whereas `this.view` is the proxied one. If `editor.isDestroyed` was based on the `editor.editorView` instead of the `editor.view`, you would not have this issue.

Reply:
I think this try catch should be reverted, as it does not even check what kind of error was thrown & may silence a real error that occured